sra_convert.py

```diff
@@ -20,66 +27,64 @@ def _parse_cmdl(cmdl):
 			"[Default: $SRARAW:" + safe_echo("SRARAW") + "]")
 
 	parser.add_argument(
-			"-m", "--mode", default="convert",
-			help="Mode. Can be: 'convert', 'delete_sra', 'delete_bam'")
-	
+			"-m", "--mode", choices=[CONVERT, DELETE_SRA, DELETE_BAM], default=CONVERT,
+			help="Subcommand action to be performed")
```

Comment:
'subcommand action to be performed' is a bit hard to understand...

Reply:
OK will change